go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/anchore/syft
 
 go 1.18
 
+replace github.com/anchore/stereoscope => github.com/deepfence/stereoscope v0.0.0-20230111090613-ee0c9713629f
```

Comment:
just a reminder for reviewers: this should be removed before merging (keep this comment unresolved for now)